### PR TITLE
Use route.href to build the link of the `main link`

### PR DESCRIPTION
**Pull request by vied12**
Because using the label to generate the link is not always relevant

### CODE REVIEW

**Comment by amagdas** [comment]
:+1:

File: client/app/scripts/superdesk/analytics/analytics.js [hunk 1]
```diff
@@ -23,7 +23,9 @@ define([
 
         .run(['$rootScope', 'analytics', function($rootScope, analytics) {
             $rootScope.$on('$routeChangeSuccess', function(ev, route) {
-                analytics.track(route.$$route);
+                if (angular.isDefined(route)) {
+                    analytics.track(route.$$route);
```
Comment (plamut) [inline client/app/scripts/superdesk/analytics/analytics.js]: I'm not sure this is the best way, attributes prefixed by `$$` are considered internal to Angular and should not be directly. They might, for instance, change without notice in future framework versions.

Is there any other way to achieve the same functionality?

**Comment by vied12** [comment]
@plamut Updated. It seems to work exactly in the same way without `$$route`.

**Comment by plamut** [comment]
Do we have a test for that? But other than that, good thing that we got rid of `$$route`.

:+1:

**Comment by vied12** [comment]
Yes there is a test, I also updated it

**Comment by plamut** [comment]
Great, feel free to merge it then.

**Comment by vied12** [comment]
I don't have the right. Please do it

**Pull request merged**